ch04packaging/03Packaging.ipynb.py

```diff
@@ -21,43 +21,57 @@
 # A good cross-platform build tool is the most important thing: you can always
 # have collaborators build from source.
 #
+# Along with the content below, the Python ecosystem has a few fantastic guides on both simple
+# and complex packaging. Some of the recommended ones are listed below -
+#  - [Python Packaging User Guide](https://packaging.python.org/en/latest/)
+#  - [pyOpenSci Python Package Guide](https://www.pyopensci.org/python-package-guide/)
+#  - [Scientific Python Packaging Guide]https://learn.scientific-python.org/development/tutorials/packaging/
+#
+# Besides the guides, there are numerous "cookie" templates available for developers. These templates
+# allow developers to generate an empty Python package following good practices and guidelines with a
+# simple CLI command.
+#  - [Scientific Python cookie](https://github.com/scientific-python/cookie)
+#  - [Open Science Labs' scicookie](https://github.com/osl-incubator/scicookie)
+#  - Domain/ecosystem specific cookies like [pybamm-cookiecutter](https://github.com/pybamm-team/pybamm-cookiecutter) for battery modeling projects in Python exist too
 
 # %% [markdown]
 # ### Distribution tools
 
 # %% [markdown]
 # Distribution tools allow one to obtain a working copy of someone else's package.
 #
-# - Language-specific tools: 
-#  - python: PyPI,
-#  - ruby: Ruby Gems, 
-#  - perl: CPAN,
-#  - R: CRAN
+#  - Language-specific tools: 
+#    - python: PyPI,
+#    - ruby: Ruby Gems, 
+#    - perl: CPAN,
+#    - R: CRAN
 #  
-# - Platform specific packagers e.g.:
-#  - [`brew`](https://brew.sh/) for MacOS, 
-#  - `apt`/`dnf`/`pacman` for Linux or 
-#  - [`choco`](https://chocolatey.org/) for Windows.
+#  - Platform specific packagers e.g.:
+#    - [`brew`](https://brew.sh/) for MacOS, 
+#    - `apt`/`dnf`/`pacman` for Linux or 
+#    - [`choco`](https://chocolatey.org/) for Windows.
 
 # %% [markdown]
 # ### Laying out a project
 
 # %% [markdown]
 #
 # When planning to package a project for distribution, defining a suitable
-# project layout is essential. A typical layout might look like this:
+# project layout is essential. A typical scientific python compliant layout
+# might look like this:
 #
 # ```
 # repository_name
-# |-- module_name
-# |   |-- __init__.py
-# |   |-- python_file.py
-# |   |-- another_python_file.py
-# |   `-- test
-# |       |-- fixtures
-# |       |   `-- fixture_file.yaml
+# |-- src
+# |   `-- package_name
 # |       |-- __init__.py
-# |       `-- test_python_file.py
+# |       |-- python_file.py
+# |       |-- another_python_file.py
+# `-- tests
+#     |-- fixtures
+#     |   `-- fixture_file.yaml
+#     |-- __init__.py
+#     `-- test_python_file.py
```

Comment:
I've never been a fun of this structure, but I know that's what all are suggesting. I don't like to have the tests far from the source as it makes it harder to follow TDD. But, if that's what all guides suggest and how the community is moving, I imagine I'll have to adapt :sweat_smile: